apps/prairielearn/assets/scripts/instructorRequestCourse.ts

```diff
@@ -0,0 +1,46 @@
+import { decodeData, onDocumentReady } from '@prairielearn/browser-utils';
+
+onDocumentReady(() => {
+  $('input[name=cr-role]').on('change', function () {
+    const role = (this as HTMLFormElement).value;
```

Comment:
Shouldn't this be `HTMLInputElement`?

```suggestion
    const role = (this as HTMLInputElement).value;
```

Reply:
I'll make the change. Either work in that they have a `value` property, but `HTMLInputElement` is more correct.